src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -2077,6 +2076,8 @@ public void DeleteWithoutCreatingTombstone(DocumentsOperationContext context, st
 
                     ThrowConcurrencyExceptionOnConflict(expectedEtag, currentMaxConflictEtag);
                 }
+
+                var fromReplication = (flags & DocumentFlags.FromReplication) == DocumentFlags.FromReplication;
```

Comment:
I didn't wrote this, just move it to the right location. It was on the beginning on the method and I moved it just before the use.